media/css/firefox/family/components/_subnav.scss

```diff
@@ -0,0 +1,170 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// * -------------------------------------------------------------------------- */
+// Should be replaced by https://github.com/mozilla/protocol/issues/471
+
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.t-firefox-family .c-sub-navigation {
```

Comment:
mostly copy/paste, changes listed:
L110 & 143, change breakpoint to 950px
L98 & 41, unset float to override conflict with protocol styles (which collapse at 768)
L121, keep display block, override conflict with protocol